scenes/player/animation_player.gd

```diff
@@ -2,15 +2,15 @@
 # SPDX-License-Identifier: MPL-2.0
 extends AnimationPlayer
 
-const FIGHT_ANIMATIONS: Array[StringName] = [&"blow_right", &"blow_left", &"blow_up", &"blow_down"]
+const BLOW_ANTICIPATION_TIME: float = 0.3
```

Comment:
Since we are using an animation player, maybe this can go in the animation track, as a track that sets the property cancellable or one that calls a method. It's usually easier to maintain since this value could change depending on the animation.

Reply:
I just realized that cancellable is not needed anymore, since I removed the first anticipation. I will add remove it and add a note in the commit message.